ast/src/parsed/mod.rs

```diff
@@ -138,6 +144,9 @@ impl PilStatement {
                         .map(move |v| (name, Some(&v.name), SymbolCategory::TypeConstructor)),
                 ),
             ),
+            PilStatement::TraitDeclaration(_, TraitDeclaration { name, .. }) => {
+                Box::new(once((name, None, SymbolCategory::TraitDeclaration)))
+            }
```

Comment:
Shouldn't this also return the trait functions?

Reply:
I don't think so, since both, `TraitImplementation` and `TraitDeclaration`, define the same `Definitions`, if functions are also included there would be collisions.

Reply:
I mean for enum it does...

Reply:
Should be fixed :)